Source/Scene/PointCloud3DTileContent.js

```diff
@@ -845,17 +833,17 @@ define([
     }
 
     function modifyStyleFunction(source) {
-        // Replace occurrences of czm_tiles3d_style_SEMANTIC with semantic
-        var length = semantics.length;
+        // Replace occurrences of czm_tiles3d_style_DEFAULTPROPERTY
+        var length = defaultProperties.length;
         for (var i = 0; i < length; ++i) {
-            var semantic = semantics[i];
-            var styleName = 'czm_tiles3d_style_' + semantic;
-            var replaceName = semantic.toLowerCase();
-            source = source.replace(new RegExp(styleName, 'g'), replaceName);
+            var property = defaultProperties[i];
+            var styleName = 'czm_tiles3d_style_' + property;
+            var replaceName = property.toLowerCase();
+            source = source.replace(new RegExp(styleName + "([^\w_$])", 'g'), replaceName + "$1");
```

Comment:
No, because after `styleName` no letter should follow. But I think I can remove `$` from it, because there should be always still a `;` after a property. And maybe I should write `\W` instead of `[^\w]`

Btw: with single quote I need to to write `\\W` instead of `\W`